ui/app/styles/components/empty-state.scss

```diff
@@ -0,0 +1,37 @@
+.empty-state {
+  align-items: center;
+  color: $grey;
+  display: flex;
+  background: $ui-gray-050;
+  justify-content: center;
+  padding: $spacing-xxl $spacing-s;
```

Comment:
Does it make sense to have separate vars for spacing and sizes or should they be the same (no strong preference here, just 🤔)?

Reply:
I think the way Bulma uses `$size-` for both is pretty weird, but the new Structure-based `$spacing-` are using a different scale, so converting those over won't be easy. I think trying to use the `$spacing-` variables for new stuff is a good idea and we can use the `$size-` variables for font sizing like it feels they were intended until we have better typography variables/styles figured out